google_guest_agent/oslogin.go

```diff
@@ -188,6 +199,55 @@ func (o *osloginMgr) Set(ctx context.Context) error {
 	return nil
 }
 
+func cleanupDeprecatedLines(fpath string, directives []string) error {
+	// If the file doesn't exist don't even try updating it.
+	stat, err := os.Stat(fpath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+		return fmt.Errorf("failed to stat config file: %+v", err)
+	}
+
+	data, err := ioutil.ReadFile(fpath)
```

Comment:
`ioutil` is deprecated, it seems you missed changing this while fixing line 232

Reply:
done. thanks.